code/modules/tiger/dildo.dm

```diff
@@ -0,0 +1,120 @@
+//Stolen from Nexendia~
+
+/obj/item/weapon/dildo
+	name = "\improper Dragon Dildo"
+	desc = "It's a Dragon Dildo. If you are seeing this, please make an issue report detailing how you came across it."
+
+	icon       = 'icons/obj/dildos.dmi'
+	icon_state = null
+
+	force      = 5
+	throwforce = 5
+	w_class    = 1
+
+	hitsound    = 'sound/items/squishy.ogg'
+	attack_verb = list("slapped")
+
+/obj/item/weapon/dildo/attack(mob/living/carbon/human/M, mob/user)
+	. = ..()
+
+	if(istype(M))
+		spawn(2)
+			if(user && M)
+				user.visible_message("<span class='userdanger'>[uppertext(user.name)] HAS ANGRERED THE GODS!</span>")
+
+				loc = get_turf(user)
+				user.gib()
+
+				M.drop_l_hand()
+				M.drop_r_hand()
+
+				M.put_in_hands(src)
+				M << "<span class='notice'>\The [src] has chosen you to be it's new master!</span><span class='danger'> Now, go fuck yourself.</span>"
+				//Yep.
+
+
+/obj/item/weapon/dildo/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is shoving \the [src] down \his throat! It looks like they're trying to commit suicide.</span>")
+	return(BRUTELOSS)
+
+/obj/item/weapon/dildo/sea
+	name = "\improper Sea Dragon Dildo"
+	desc = "A Sea Dragon Dildo. Always wet!"
+
+	icon_state = "seadragon"
+
+	w_class = 2
+
+
+/obj/item/weapon/dildo/canine
+	name = "\improper Vulvakin Dildo"
+	desc = "A Canine Dildo, designed to look very close to those of the Vulvakin species."
+
```

Comment:
I assure you not, good sir @iamgoofball! The white space is completely intentional, to enhance the length and girth of the Tiger Dildo Emporium&trade;'s many dildos!